doc/supported-hardware.md

```diff
@@ -5,6 +5,7 @@ The following SR-IOV capable hardware is supported with sriov-network-operator:
 | Model                    | Vendor ID | Device ID |
 | ------------------------ | --------- | --------- |
 | Intel XXV710 Family |  8086 | 158b |
+| Intel X557 Family |  8086 | 1589 |
```

Comment:
There is a second table below in the "Supported features per hardware" section. Would you be able to share what is working for you? I think "SR-IOV Kernel" and "SR-IOV DPDK" work fine but input from you would be great.

Reply:
done!